src/Components/Address/__tests__/useAddressAutocomplete.jest.ts

```diff
@@ -41,12 +43,17 @@ describe("useAddressAutocomplete", () => {
 
   afterAll(() => {
     jest.resetAllMocks()
+    jest.resetModules()
```

Comment:
What does this get us?

Reply:
`jest.resetModules()` resets the module registry and clears all cached module imports so that you get a fresh copy of the imported module in each test. It's useful for when a module has internal state that you don't want persisted. I was experimenting with it when I thought there was a state leakage problem happening between the tests or some kind of side effects causing other tests to fail. 

The test still pass without it though, so I might just remove it.